tv.kodi.Kodi.yml

```diff
@@ -27,18 +27,18 @@ command: kodi
 rename-desktop-file: kodi.desktop
 rename-icon: kodi
 modules:
-  - shared-modules/udev/udev-175.json
+  - shared-modules/intltool/intltool-0.51.json
   - shared-modules/python2.7/python-2.7.json
-  - shared-modules/glu/glu-9.0.0.json
+  - shared-modules/glu/glu-9.json
   - deps/avahi.json
   - deps/libical.json
   - deps/bluez.json
   - deps/fmt.json
-  - deps/fribidi.json
   - deps/fstrcmp.json
   - deps/libass.json
   - deps/libcdio.json
   - deps/lzo2.json
+  - deps/libtirpc.json
```

Comment:
Why is libtirpc added?

Reply:
It's needed by samba after glibc dropped rpc headers, see [commit description](https://github.com/flathub/tv.kodi.Kodi/pull/69/commits/60b557d118ec764e13a63401d7dec4d5b5f2d65c).